yellowstone-grpc-geyser/src/grpc.rs

```diff
@@ -838,6 +839,13 @@ impl GrpcService {
                         }
                     }
                     message = messages_rx.recv() => {
+                        let num_slots_behind = NUM_SLOTS_BEHIND.load(Ordering::SeqCst);
+                        if num_slots_behind > HEALTH_CHECK_SLOT_DISTANCE {
```

Comment:
Not required in first release, but I would prefer if this is configured as two checks:
1. Auto-disconnect is >100 slots
2. Disconnect if >20 slots for last 5 checks